ui/helpers/utils/confirm-tx.util.ts

```diff
@@ -95,8 +117,10 @@ export function roundExponential(decimalString) {
     : decimalString;
 }
 
-export function areDappSuggestedAndTxParamGasFeesTheSame(txData = {}) {
-  const { txParams, dappSuggestedGasFees } = txData;
+export function areDappSuggestedAndTxParamGasFeesTheSame(
+  txData: TransactionMeta,
+) {
```

Comment:
would be great to add boolean as return type here ~

Reply:
Done in latest